### ISSUE TITLE
Is the Photo Frame Mode working?

### ISSUE THREAD

**Issue by TimoD87** [pookjw/StandPad#1]
I would directly buy an iPad, which is supported by the jailbreak if you can confirm, that the foto frame feature is working :-) 

There is no this good photoframe in the App Store and also not with the native photo app.

Looking forward to hear from you, greetings from Germany

**Comment by pookjw** [comment]
No, Photo and Large Clock Widgets are not supported yet. I've tried, but I don't know how to enable them.

**Comment by TimoD87** [comment]
Any chance that you look deeper into it? 🙏 
As I said in my opinion, there is no photo frame on the Apple ecosystem which could come close to the standby Photo Frame used on an iPad. 

And thanks for your reply!!!

**Comment by TimoD87** [comment]
Push 😊
